CellProfiler_pipelines/README.md

```diff
@@ -0,0 +1,107 @@
+
+Based on my understanding of CellProfiler, these modules identify the objects and create masks for each of them to use for feature extraction.
+
+### Feature Extraction
+
+By using the object masks, each module runs through each image from every channel to extract features from.
+I followed the format of one of the CP pipelines from the [Broad Institute Platform Pipelines](https://github.com/broadinstitute/imaging-platform-pipelines/blob/master/cellpainting_UMUC9_20x_phenix_bin2/analysis_without_batchfile.cppipe).
+The following modules were used:
+
+- `MeasureColocalization`
+
+- `MeasureGranularity`
+
+- `MeasureImageIntensity`
+
+- `MeasureObjectNeighbors`
+
+- `MeasureObjectIntensityDistribution`
+
+- `MeasureObjectSizeShape`
+
+- `MeasureTexture`
+
+For further information on each of these modules, see the [Measurement section](https://cellprofiler-manual.s3.amazonaws.com/CellProfiler-4.2.1/modules/measurement.html) of the CellProfiler manual.
+
+## Step 1: Setup CellProfiler Environment
+
+### Step 1a: Create Environment
+ 
+ I used this [.yml file format](https://github.com/CellProfiler/CellProfiler/wiki/Conda-Installation) from the Wiki. I updated the file and the Wiki page so that it would pip install the newest verison of CellProfiler instead of the older version that it had before.
+
+ ```sh
+# Run this command in terminal to create the conda environment for CellProfiler
+conda env create -f download_cellprofiler.yml
+```
+
+### Step 1b: Activate Environment
+
+```sh
+# Run this command in terminal to activate the conda environment for CellProfiler
+conda activate cp4
+```
+
+## Step 2: Open CellProfiler
+
+```sh
+# Run this command in terminal to start CellProfiler
+cellprofiler
+```
+
+## Step 3: Run `illum.cpproj` and `analysis.cpproj`
+
+`illum.cpproj` will be run first before running `analysis.cpproj`.
+The same steps in order below will be followed for each project.
+
+When CellProfiler is open, you will drag the CP project file from your file explorer into the box on the CellProfiler interface, as shown below:
+
+![CellProfiler_fig1.png](Example_Images/CellProfiler_fig1.png)
+
+After the project is loaded in, it will look like the image below:
+
+![CellProfiler_pipeline.png](Example_Images/CellProfiler_pipeline.png)
+
+Make sure that the `Images` module contains all of the NF1 images like the above image shows before proceeding.
+To run the illumination correction pipeline, press `Analyze Images` at the bottom right.
+
+**Note:** If you do not want to see the pop ups from each module in the pipeline, you can have the `eye symbol` turned off, like seen in the image above for all modules underneath `IdentifyTeritaryObjects`. 
+This is done by just pressing that symbol to turn it on or off depending on what modules you want to view.
+All of the segmentation modules are turned on as to view how the parameters are working.
```

Comment:
What do you mean by "how the parameters are working"? Can you clarify? Did you test different parameters? (I think I asked this question above)

Reply:
Fixed the wording!